stdlib/range.gr

```diff
@@ -0,0 +1,15 @@
+// TODO: Exclusive
+enum Range {
+  Inclusive(Number, Number),
+}
+
+export let inclusive = (lower, upper) => {
+  Inclusive(lower, upper)
+}
+
+export let isBetween = (value, range) => {
```

Comment:
Should this be data last?

Reply:
It is! The "range" is the data because it's the range module

Reply:
Oh interesting. I hadn't really thought of ranges as data, since I'd usually want to test some other data against the range. I see myself wanting to do stuff like `List.filter(partial Range.isBetween(myRange), myList)`. I can't really argue with your logic, though. I'll happily trust your judgment on what to do here though 👍

Reply:
I definitely think your example will be a common case. I was thinking it would be achieved with placeholder values like `List.filter(partial Range.isBetween(_, myRange), myList)` but maybe that is too verbose?

Reply:
As I look at this again, I wonder if this should be called `inRange`?

Reply:
Renamed! I'm not particular and `inRange` makes the range-last make more sense.